crates/astria-core/src/sequencerblock/v1alpha1/block.rs

```diff
@@ -1439,6 +1376,12 @@ impl Deposit {
     }
 }
 
+impl From<Deposit> for crate::generated::sequencerblock::v1alpha1::Deposit {
```

Comment:
Not new, just moved down from above; this is an artifact of the rafctor, but the trait impl fits better down here.